contracts/payloads/IGP9/PayloadIGP9.sol

```diff
@@ -0,0 +1,335 @@
+
+    /// @notice                   Sets an address as allowed user or not. Only callable by auths.
+    /// @param user_              address to set allowed value for
+    /// @param allowed_           bool flag for whether address is allowed as user or not
+    function setUserAllowed(address user_, bool allowed_) external;
+
+    /// @notice                   Sets `maxLTV` to `maxLTV_` (in 1e2: 1% = 100, 100% = 10000). Must be > 0 and < 100%.
+    function setMaxLTV(uint16 maxLTV_) external;
+
+    /// @notice                   Sets an address as allowed guardian or not. Only callable by owner.
+    /// @param guardian_          address to set guardian value for
+    /// @param allowed_           bool flag for whether address is allowed as guardian or not
+    function setGuardian(address guardian_, bool allowed_) external;
+}
+
+contract PayloadIGP9 {
+    uint256 public constant PROPOSAL_ID = 9;
+
+    address public constant PROPOSER =
+        0xA45f7bD6A5Ff45D31aaCE6bCD3d426D9328cea01;
+
+    IGovernorBravo public constant GOVERNOR =
+        IGovernorBravo(0x0204Cd037B2ec03605CFdFe482D8e257C765fA1B);
+    ITimelock public constant TIMELOCK =
+        ITimelock(0x2386DC45AdDed673317eF068992F19421B481F4c);
+
+    address public constant TEAM_MULTISIG = 
+        0x4F6F977aCDD1177DCD81aB83074855EcB9C2D49e;
+
+    address public constant LITE =
+        0xA0D3707c569ff8C87FA923d3823eC5D81c98Be78;
+    address public constant LITE_DSA = 
+        0x9600A48ed0f931d0c422D574e3275a90D8b22745;
+
+    address public immutable ADDRESS_THIS;
+    
+    IstETHProtocol public constant STETH_PROTOCOL = IstETHProtocol(0x1F6B2bFDd5D1e6AdE7B17027ff5300419a56Ad6b);
+    IFluidLiquidityAdmin public constant LIQUIDITY = IFluidLiquidityAdmin(address(0)); // TODO
+
+    address public constant ETH_ADDRESS = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
+
+    constructor() {
+        ADDRESS_THIS = address(this);
+    }
+
+    function propose(string memory description) external {
+        require(msg.sender == PROPOSER, "msg.sender-not-proposer");
+
+        uint256 totalActions = 1;
+        address[] memory targets = new address[](totalActions);
+        uint256[] memory values = new uint256[](totalActions);
+        string[] memory signatures = new string[](totalActions);
+        bytes[] memory calldatas = new bytes[](totalActions);
+
+        // Action 1: call executePayload on timelock contract to execute payload related to Fluid
+        targets[0] = address(TIMELOCK);
+        values[0] = 0;
+        signatures[0] = "executePayload(address,string,bytes)";
+        calldatas[0] = abi.encode(ADDRESS_THIS, "execute()", abi.encode());
+
+        uint256 proposedId = GOVERNOR.propose(
+            targets,
+            values,
+            signatures,
+            calldatas,
+            description
+        );
+
+        require(proposedId == PROPOSAL_ID, "PROPOSAL_IS_NOT_SAME");
+    }
+
+    function execute() external {
```

Comment:
Would be nice to restrict execute if when calling `address(this) == timelock`

Reply:
okay!